terraform/docker-chef.sh

```diff
@@ -227,11 +235,10 @@ docker_svc_start () {
   echo "Ensuring $dirs directories exist and removing stale LOCK files"
   mkdir -p $dirs
   rm -f ${DATA_MOUNT:-/mnt/hab}/${1}_sup/default/LOCK
-  $(sudo_cmd) docker run --rm -it \
+  $(sudo_cmd) docker run --rm -i \
```

Comment:
I'm concerned this will break the `/dev/stdout` logging we have in Nginx -  I thought we needed the tty allocation just for that?

Reply:
I thought so too at first, however I can confirm that without the `-t` Nginx starts normally, with no emergency errors, and logging works as normal:

```
chef-server-nginx.default(O): 172.31.28.151 - - [09/Aug/2018:17:40:39 +0000]  "POST /users/ HTTP/1.1" 201 "0.613" 1767 "-" "Chef Knife/12.17.47 (ruby-2.4.2-p198; ohai-8.22.0; x86_64-linux; +https://chef.io)" "172.31.28.151:8000" "201" "0.612" "12.17.47" "algorithm=sha1;version=1.1;" "pivotal" "2018-08-09T17:40:38Z" "JqBbYln+D8ge9LKXVV4sGasMbO8=" 1212
chef-server-nginx.default(O): 127.0.0.1 - - [09/Aug/2018:17:40:43 +0000]  "GET /_status HTTP/1.1" 200 "0.006" 243 "-" "curl/7.54.1" "172.31.28.151:8000" "200" "0.006" "-" "-" "-" "-" "-" 85
chef-server-nginx.default hook[health_check]:(HK): {"status":"pong","upstreams":{"chef_elasticsearch":"pong","chef_sql":"pong","oc_chef_authz":"pong"},"keygen":{"keys":10,"max":10,"max_workers":10,"cur_max_workers":10,"inflight":0,"avail_workers":10,"start_size":0},"indexing":{"mode":"batch"}}
```